### DISCUSSION TITLE
publish UTM on appdb

### DISCUSSION THREAD

**Discussion by dbs2022** [utmapp/UTM#6578] [General]
Hello! We are inviting you to publish UTM on appdb officially.

Appdb is an app store that works independently from Apple, but is available on Apple-branded devices.

https://appdb.to/news/529

**Comment by osy** [comment]
Sorry for the late response, but what do we have to do for this? @dbs2022

**Reply by dbs2022** [reply]
Hi! We have published developer journey https://rtfm.dbservices.to/#/developer-apps-publishing/journey
It will take about hour of your time, if you have any questions, we will be glad to help.
